structuredheadings/plugin.js

```diff
@@ -18,8 +55,31 @@
   var firstHeaderKey = 0;
   var lastHeaderKey = headerList.length - 1;
 
+//style helpers
+
+
+  var clearStyles = function (element) {
+    for (var styleName in config.autonumberStyles) {
+      var style = config.autonumberStyles[styleName];
+      for (var className in style) {
+        element.removeClass(style[className]);
+      }
+    }
+  };
+
+  var setStyle = function (element, styleName) {
+    var elementType = element.getName();
+    var style = config.autonumberStyles[styleName];
+    if (elementType) {
```

Comment:
I left that check out, if the element isn't a heading, the style check will fail since there is no style setup in the config for that element, so no style is changed. So setStyle could be used on a non-Header element. But in the current form, the function is only being applied to elements that have already run through the .is(allowedElements) check.